tck/metrics/src/main/java/org/eclipse/microprofile/telemetry/metrics/tck/http/HttpHistogramTest.java

```diff
@@ -0,0 +1,185 @@
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.sdk.autoconfigure.spi.metrics.ConfigurableMetricExporterProvider;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.ApplicationPath;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.core.Application;
+import jakarta.ws.rs.core.Response;
+
+public class HttpHistogramTest extends Arquillian {
+
+    private static final AttributeKey<String> HTTP_REQUEST_METHOD = AttributeKey.stringKey("http.request.method");
+    private static final AttributeKey<String> URL_SCHEME = AttributeKey.stringKey("url.scheme");
+    private static final AttributeKey<String> HTTP_RESPONSE_STATUS_CODE =
+            AttributeKey.stringKey("http.response.status_code");
+    private static final AttributeKey<String> NETWORK_PROTOCOL_NAME =
+            AttributeKey.stringKey("network.protocol.name");
+    private static final AttributeKey<String> HTTP_ROUTE = AttributeKey.stringKey("http.route");
+    private static final AttributeKey<String> ERROR_TYPE = AttributeKey.stringKey("error.type");
+
+    @Inject
+    OpenTelemetry openTelemetry;
+
+    @Deployment
+    public static WebArchive createTestArchive() {
+        return ShrinkWrap.create(WebArchive.class)
+                .addClasses(InMemoryMetricExporter.class, InMemoryMetricExporterProvider.class,
+                        BasicHttpClient.class)
+                .addAsLibrary(TestLibraries.AWAITILITY_LIB)
+                .addAsServiceProvider(ConfigurableMetricExporterProvider.class, InMemoryMetricExporterProvider.class)
+                .addAsResource(new StringAsset(
+                        "otel.sdk.disabled=false\notel.metrics.exporter=in-memory\notel.logs.exporter=none\notel.traces.exporter=none\notel.metric.export.interval=3000"),
+                        "META-INF/microprofile-config.properties")
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
+    }
+
+    @ArquillianResource
+    private URL url;
+    @Inject
+    private InMemoryMetricExporter metricExporter;
+
+    private BasicHttpClient basicClient;
+
+    @BeforeMethod
+    void setUp() {
+        if (metricExporter != null) {
+            metricExporter.reset();
+            basicClient = new BasicHttpClient(url);
+        }
+    }
+
+    @Test
+    void testHTTPMetricAttributes() {
```

Comment:
should also test for name (`http.server.request.duration`) / description / type / unit for HTTP metrics